src/autocomplete/databases/trino/grammar/TrinoParser.g4

```diff
@@ -7,110 +7,110 @@ options {
     tokenVocab = TrinoLexer;
 }
 
+// TODO-TRINO: support multi-queries
 parse
-    : statements? EOF
-    ;
-
-statements
-    : statement SEMICOLON_?
-    | statement SEMICOLON_ statements
+    : statement? EOF
     ;
 
 statement
-    : rootQuery                                          # statementDefault
-    | USE_ schema = identifier                           # use
-    | USE_ catalog = identifier DOT_ schema = identifier # use
-    | CREATE_ CATALOG_ (IF_ NOT_ EXISTS_)? catalog = identifier USING_ connectorName = identifier (
```

Comment:
Basically I've splitted a bunch of simple identifiers to `tableIdentifier`, `schemaIdentifier`, `catalogIdentifier` to have a simple autocomplete